lib/PuppeteerSharp.Tests/Page/SetJavaScriptEnabledTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Page
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class SetJavaScriptEnabledTests : PuppeteerBaseTest
+    {
+        [Fact]
+        public async Task ShouldWork()
+        {
+            var page = await Browser.NewPageAsync();
+            await page.SetJavaScriptEnabledAsync(false);
+            await page.GoToAsync("data:text/html, <script>var something = 'forbidden'</script>");
+
+            var exception = await Assert.ThrowsAnyAsync<Exception>(async () => await page.EvaluateFunctionAsync("something"));
```

Comment:
not sure why I had to execute `EvaluateFunctionAsync` here but it only worked this way (line22 uses `EvaluateExpressionAsync`)

Reply:
I think we need to test `EvaluateExpressionAsync` here. If `EvaluateFunctionAsync` failed it's maybe because that's not a valid function call. But we definitely have to try `EvaluateExpressionAsync` here as Puppeteer does.